test/views/IndexViewSpec.scala

```diff
@@ -282,7 +282,7 @@ class IndexViewSpec extends SpecBase with ViewBehaviours {
         "populate summary card when user has LPPs" in {
           vatTraderDoc.select(Selectors.summaryCardHeaderTitle(Selectors.summaryLPPCard)).text shouldBe lppHeader
           vatTraderDoc.select(Selectors.viewCalculation).text shouldBe viewCalculationLink
-          vatTraderDoc.select(Selectors.viewCalculation).attr("href") shouldBe "/penalties/calculation?penaltyId=123456789"
```

Comment:
can please we append the similar test in "populate summary card when user has LPPs with additional penalties"

vatTraderDocLPPAdditionalPenalty.select(Selectors.viewCalculation).text shouldBe viewCalculationLink
          vatTraderDocLPPAdditionalPenalty.select(Selectors.viewCalculation).attr("href") shouldBe "/penalties/calculation?penaltyId=123456789&isAdditional=true"